files/pull-image.sh

```diff
@@ -19,6 +19,11 @@ function retry() {
   done
 }
 
+### Short-circuit fetching sandbox image if its already present
+if [[ "$(sudo ctr --namespace k8s.io image ls | grep $sandbox_image)" != "" ]]; then
```

Comment:
I think you actually want to put this in https://github.com/awslabs/amazon-eks-ami/blob/1339eda7f7a811b2fbc7ce0d5966635cf0855bf7/files/pull-sandbox-image.sh because I'm not sure we want this behavior for all images passed to `pull-image.sh`. 

(also the variable has changed in this script from `sandbox_image` to `img` after #938 )